src/components/UI/Selector/Selector.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
```

Comment:
Sorry it's totally too late to give this feedback, but I found that it's quite unique that you have `UI` folder within a component folder for global components 😄